### ISSUE TITLE
 feature-request: add css-class to events

### ISSUE THREAD

**Issue by Heklaterriol** [jemproject/JEM-Project#1754]
Would be nice to add this feature.
I suggest that the class would be added to the body tag for maximal range of possible styling.

**Comment by Heklaterriol** [comment]
This was a request in the forum.
Ther is an old branch of mine where it was partly realized and I thought, that I merged it in to the dev branch at that time – but obviously I didn't.
I will check it out.

**Comment by Heklaterriol** [comment]
I added some classes, see the commits here #1601 or the branch and test build here:
https://github.com/jemproject/JEM-Project/tree/id-class-for-event/venue/category

I added classes to neary everything;
`event_id, venue_id, cat_id`  always with the according ID number, like `event_id1, event_id4, cat_id7` etc.

Please test!

As a test color scheme you can use this one:

```
.event_id1 {background-color: #FF6B6B !important; }
.event_id2 {background-color: #4ECDC4 !important; }
.event_id3 {background-color: #45B7D1 !important; }
.event_id4 {background-color: #FFA07A !important; }
.event_id5 {background-color: #98D8C8 !important; }
.event_id6 {background-color: #F7DC6F !important; }
.event_id7 {background-color: #BB8FCE !important; }
.event_id8 {background-color: #F1948A !important; }

.cat_id1 {background-color: #82E0AA !important; }
.cat_id2 {background-color: #F8C471 !important; }
.cat_id3 {background-color: #85C1E9 !important; }
.cat_id4 {background-color: #F1C40F !important; }
.cat_id5 {background-color: #D7BDE2 !important; }
.cat_id6 {background-color: #73C6B6 !important; }
.cat_id7 {background-color: #F0B27A !important; }
.cat_id8 {background-color: #C39BD3 !important; }

.venue_id1 {background-color: #7FB3D5 !important; }
.venue_id2 {background-color: #E59866 !important; }
.venue_id3 {background-color: #76D7C4 !important; }
.venue_id4 {background-color: #F7DC6F !important; }
.venue_id5 {background-color: #EC7063 !important; }
.venue_id6 {background-color: #A9DFBF !important; }
.venue_id7 {background-color: #D2B4DE !important; }
.venue_id8 {background-color: #FAD7A0 !important; }
```

**Comment by mckillo** [comment]
I applied the commits and in my test, all views worked, and nothing broken. Good job, for me, it will merge to JEM-4.2.3-dev branch.

**Comment by Heklaterriol** [comment]
Thanks for testing and for your feedback. 

I just thought that we need a more specific class naming according to the view. 
Now the classes are always the same in every view, always event_id, cat_id, venue_id, each plus the id number.
I think it would be good to have a more specific class naming, because otherwise it's hard to catch them precisely, for example, if you only want to style the single event view. 

I will make a suggestion the next days.

**Comment by mckillo** [comment]
I think that this issue can be closed because it already applies to 4.2.3
